__tests__/app.test.js

```diff
@@ -74,19 +74,81 @@ describe("Get /api/articles/:article_id", () => {
     return request(app)
```

Comment:
I would like to see the sad path testing in the same describe block as the happy path for all endpoints

So sad path get /api/articles:article_id test blocks need to fall under the describe on line 52 as opposed to loads of the same describe blocks

Reply:
(again really sorry for going back to a previously approved branch 😬 classic be review week shenanigans)